xprom/xkafkaprom/collector.go

```diff
@@ -0,0 +1,230 @@
+package xkafkaprom
```

Comment:
Should we make the package name simply as `xprom/xkafka` rather than `xprom/xkafkaprom`?

Reply:
i want to avoid the name conflict while creating and configuring a producer/consumer.

Maybe we can use a convention like `xprom*` -  `xpromkafka`, `xpromdb`, `xpromsql` etc.

Reply:
i have noticed this pattern in https://github.com/riverqueue/river